test/fixtures.test.ts

```diff
@@ -2,8 +2,9 @@ import path from 'node:path'
 
 import { ESLint } from 'eslint'
 
-const getCli = (lintCodeBlocks = false) =>
+const getCli = (lintCodeBlocks = false, fix?: boolean) =>
```

Comment:
I added `fix` cases in this PR to see whether `fix` feature is working as expected.